editor.planx.uk/src/pages/FlowEditor/lib/__tests__/automations.test.ts

```diff
@@ -1,192 +0,0 @@
-import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
```

Comment:
This outdated test suite is removed in favor of new files: 
- `automations.parentChild.test.ts`
- `automations.blanks.test.ts`
- `preview/autoAnswerableOptions.test.ts`